### DISCUSSION TITLE
Compile error

### DISCUSSION THREAD

**Discussion by Velkumar10** [ptc-iot-sharing/BMCollectionView#42] [Q&A]
Hi,

I'm trying to build this extension but I'm getting the below error :

![image](https://user-images.githubusercontent.com/29228894/136150824-7647c5ac-3db0-4a31-87ba-6469ae138a96.png)

Could you please help me to fix this ?

**Comment by BogdanMihaiciuc** [comment]
Hey, this happens because I mistakenly pushed an update to the master branch where I was using a reference to my local core ui type definitions instead of the ones from `node_modules`. I will correct this on the master branch soon, but in the meantime you should be able to resolve this by decommenting the first line in `ide.ts` and commenting the second one.

**Reply by Velkumar10** [reply]
I did that still I'm getting an error

![image](https://user-images.githubusercontent.com/29228894/136188370-6c0e4b98-0a37-4d26-a093-1ac138b0d601.png)

**Reply by BogdanMihaiciuc** [reply]
Also update the `bm-core-ui` dependency to the latest version (`2.7.0`) and rerun npm install.
